docs/rundown.rst

```diff
@@ -228,3 +307,17 @@ stable:
      - Signing process failure
      - * The signer's private key was not found in the GPG keyring
        * The wrong GnuPG home directory or keyring file was specified
+
+Other Helpful Commands:
+======================
+``$ time ansible-sign project gpg-sign .``
+This command shows how long validation took.
+
+
+
+Additional Resources:
+====================
+
+.. _AWX project sync via action plugin: https://www.youtube.com/watch?v=srajyQpzkmI 
+.. _Ansible-sign CLI: https://www.youtube.com/watch?v=fyb0OxejYnk
```

Comment:
I don't think I would include these. They were meant to be internal demos and also some things have changed since recording them.